guides/doc-Installing_Proxy/master.adoc

```diff
@@ -26,15 +26,6 @@ include::common/assembly_installing-capsule-server.adoc[leveloffset=+1]
 
 include::common/assembly_performing-additional-configuration-on-smart-proxy-server.adoc[leveloffset=+1]
 
-// Configuring {SmartProxyServer} with External Services
-include::common/assembly_configuring-external-services.adoc[leveloffset=+1]
-
-include::common/assembly_managing-dhcp-on-smart-proxies.adoc[leveloffset=+1]
-
-include::common/assembly_managing-dns-on-smart-proxies.adoc[leveloffset=+1]
-
-include::common/assembly_using-infoblox-as-dhcp-and-dns-providers.adoc[leveloffset=+1]
-
 :numbered!:
 
 // {SmartProxyServer} Scalability Considerations
```

Comment:
Below there are 2 appendices that talk about DHCP settings. I'd move those over to the new guide as well.

Reply:
I moved them to the new guide.
Should we keep them as appendixes in the DHCP chapter? Based on the content they contain, it would make sense.